src/Microsoft.VisualStudio.Threading.Analyzers/VSTHRD011UseAsyncLazyAnalyzer.cs

```diff
@@ -57,7 +67,23 @@ private void AnalyzeNode(SyntaxNodeAnalysisContext context)
                     && typeArg.BelongsToNamespace(Namespaces.SystemThreadingTasks);
                 if (typeArgIsTask)
```

Comment:
Seems to be an extra tab after this line (hard to tell in diff view)

Reply:
I don't see one.